optics/src/Optics.hs

```diff
@@ -847,6 +850,94 @@ import Data.Either.Optics                    as P
+-- |               __'Getter's__                                            |
+-- +--------------------+---------------------------------------------------+
+-- | 'to'               | Build a 'Getter' from a plain function.           |
+-- +--------------------+---------------------------------------------------+
+-- | 'view' / '^.'      | View a single target.                             |
+-- +--------------------+---------------------------------------------------+
+-- | 'views'            | View after applying a function.                   |
+-- +--------------------+---------------------------------------------------+
+-- |              __'Setter's__                                             |
+-- +--------------------+---------------------------------------------------+
+-- | 'sets'             | Build a 'Setter' from an update function.         |
+-- +--------------------+---------------------------------------------------+
+-- | 'mapped'           | Build a 'Setter' from the 'Functor' class.        |
+-- +--------------------+---------------------------------------------------+
+-- | 'set' / '.~'       | Replace target(s) with value.                     |
+-- +--------------------+---------------------------------------------------+
+-- | 'over' / '%~'      | Replace target(s) by applying a function.         |
+-- +--------------------+---------------------------------------------------+
+-- |              __'Fold's__                                               |
+-- +--------------------+---------------------------------------------------+
+-- | 'folded'           | Build a 'Fold' from the 'Foldable' class.         |
+-- +--------------------+---------------------------------------------------+
+-- | 'toListOf' / '^..' | Return a list of the target(s).                   |
+-- +--------------------+---------------------------------------------------+
+-- |              __'Traversal's__                                          |
+-- +--------------------+---------------------------------------------------+
+-- | 'traversed'        | Build a 'Traversal' from the 'Traversable' class. |
+-- +--------------------+---------------------------------------------------+
+-- | 'traverseOf'       | Update target(s) with an 'Applicative'.           |
+-- +--------------------+---------------------------------------------------+
+-- |              __'Prism's__                                              |
+-- +--------------------+---------------------------------------------------+
+-- | 'prism'            | Build a 'Prism' from a constructor and matcher.   |
+-- +--------------------+---------------------------------------------------+
+-- | 'review' / '#'     | Use a 'Prism' to construct the sum type.          |
+-- +--------------------+---------------------------------------------------+
+-- | 'preview' / '^?'   | Match the target or return 'Nothing'.             |
```

Comment:
It might be good to create a new section for `AffineFold` and move preview/previews there (same for `IxAffineFold` for ipreview/ipreviews).

Reply:
Done, thanks.